### ISSUE TITLE
Edge Store

### ISSUE THREAD

**Issue by BuzzKnightbeer** [kakaroto/Beyond20#1052]
Hi, I know your experience with the Edge team has been shit in the past and I agree that their method of posing as a user to get you to add the extension into their store is horrible. It would still be nice if you could reconsider your stance on publishing there as it would make it nicer for us Edge users. Putting your personal quabble with them before that feels kinda weird to be honest.

**Comment by Aeristoka** [comment]
Alright, a bit of clarification:

They didn't pose as a user to get the extension on the store. A malicious user cloned Beyond20, built it with Malware, and submitted it to the Edge store.

How would it be nice to Edge users precisely?

Edge is Chromium at the core. You can install any Chrome Store extension with absolutely 0 issues. What would be helped by the extension being on the Edge store?

I ask @kakaroto to add any comments he finds necessary also.

**Comment by BuzzKnightbeer** [comment]
I'm talking about this ticket: #796 

It would be nicer in a way that users don't have to check the 'allow other store' option, everything would be consolidated in one extension section and not split up between edge extensions and third-party extensions, and I think it would make it easier for user that are less knowledgeable with going to the Chrome store.

I don't know the process, but as both browsers run on Chromium, I would guess bringing it into the Edge store is very straight forward?

**Comment by kakaroto** [comment]
Hey @BuzzKnightbeer,
This isn't about a personal quabble but rather about a waste of limited resources. Edge is nothing special, it's a Chromium browser like 100 others, and there is no reason for me to create, manage, publish, etc.. on the Edge store when it's already on the Chrome store. If I did that, then why not also do it for the Brave store, and the Opera store, and the 100 other browsers that are based on Chrome which already support extensions from the Chrome store. Doing a release already takes me about 2 hours of work, if I have to login into 10 different sites and submit, and pass review and verifications and all of that, every time, it's an extremely time consuming and wasteful thing to do. I support the Chrome and Firefox stores because those are two different things, the Mozilla backend/browser and the Chromium backend/browser.

Being on the Edge store is nothing more than Microsoft trying to brand itself separately from Google, and I'm not going to be the one stuck in between a fight of two giants.
As for the "Allow other store" option, that's again nothing more than Microsoft's strategy to separate itself from Chrome, while at the same time, using 99.9% of the source code from Chrome. If they don't like Chrome, then they should build their own browser instead of copying it, changing its name, and then saying the Chrome store is "another store". (oh wait, they did try that, and everyone knows how much Internet Explorer was a failure). Regardless, I'd consider that option to be more of an issue with the Edge browser itself, rather than my own problem.

My squabble with Edge in issue #796 was that they were sending me multiple emails per week, even after I told them multiple times that I was not interested in adding extra overhead for me for zero advantage, and they kept spamming me, and then creating that issue on the issue tracker. It's not the reason why I'm not adding the extension to the Edge store, it's just why I didn't like seeing that specific issue created. The reason, as I explained above, is the extra overhead and honestly, I don't have enough time to just live my life, I'm not going to waste the little free time I have left to breath, to instead doing redundant things.

When there was the malicious person who submitted a cloned copy of Beyond20 with malware to the Edge store though, I did consider doing it, as that made it (finally) a good reason to submit it there too, but then I just forgot about it.
I'll see if I can do it for the next release, though if it's some complicated process that takes hours of work again, I'd probably give up on it. I'm also quite worried about opening the floodgates and then being asked to do on the stores of every other clone of Chromium.

Thanks for understanding.
